README.md

```diff
@@ -31,7 +32,7 @@ Nanos in virtualbox on a mac but it will be slow and hard to configure.
 
 You can build and run on mac and linux. Nanos supports KVM on linux and
 HVF on osx currently for acceleration. [OPS](https://ops.city) has facilities to deploy to
```

Comment:
Should OPS be lowercase here to conform to the other references?

Reply:
OK, I replaced all uppercase OPS occurrences with lowercase